tests/test_real.py

```diff
@@ -257,8 +428,149 @@ def test_noise():
     cf_2 = cf_2.withVariance(var_2)
     assert cf_1==cf_2,'Inconsistent noise properties from getNoise and getNoiseProperties'
 
+
+@timer
+def test_normalize_area():
+    """Check that normalize_area works as expected"""
+    f606w_cat = galsim.RealGalaxyCatalog('AEGIS_F606w_catalog.fits', dir=image_dir)
+    f814w_cat = galsim.RealGalaxyCatalog('AEGIS_F814w_catalog.fits', dir=image_dir)
+
+    psf = galsim.Gaussian(fwhm=0.6)
+
+    rng = galsim.BaseDeviate(5772)
+    crg1 = galsim.ChromaticRealGalaxy([f606w_cat, f814w_cat], random=True, rng=rng.duplicate())
+    crg2 = galsim.ChromaticRealGalaxy([f606w_cat, f814w_cat], random=True, rng=rng.duplicate(),
+                                      normalize_area=True)
+    assert crg1 != crg2
+    LSST_i = galsim.Bandpass(os.path.join(bppath, "LSST_r.dat"), 'nm')
+    obj1 = galsim.Convolve(crg1, psf)
+    obj2 = galsim.Convolve(crg2, psf)
+    im1 = obj1.drawImage(LSST_i, exptime=1, area=1)
+    im2 = obj2.drawImage(LSST_i, exptime=1, area=galsim.real.HST_area)
+    printval(im1, im2)
+    np.testing.assert_array_almost_equal(im1.array, im2.array)
+    np.testing.assert_almost_equal(
+            obj1.noise.getVariance(),
+            obj2.noise.getVariance() * galsim.real.HST_area**2)
+
+    rg1 = galsim.RealGalaxy(f606w_cat, index=1)
+    rg2 = galsim.RealGalaxy(f606w_cat, index=1, normalize_area=True)
+    assert rg1 != rg2
+    obj1 = galsim.Convolve(rg1, psf)
+    obj2 = galsim.Convolve(rg2, psf)
+    im1 = obj1.drawImage()
+    im2 = obj2.drawImage(exptime=1, area=galsim.real.HST_area)
+    printval(im1, im2)
+    np.testing.assert_array_almost_equal(im1.array, im2.array)
+    np.testing.assert_almost_equal(
+            obj1.noise.getVariance(),
+            obj2.noise.getVariance() * galsim.real.HST_area**2)
+
+
+@timer
+def test_crg_noise_draw_transform_commutativity():
+    """Test commutativity of ChromaticRealGalaxy correlated noise under operations of drawImage and
+    applying transformations.
+    """
```

Comment:
I more-or-less copied `test_CRG_noise.py` into an actual test, and used an `if __name__ == '__main__'` block to isolate a fastish version (~6 sec on my laptop) from some more interesting versions (~60 sec).